### PR TITLE
Removes growth serum

### PR DESCRIPTION
**Pull request by Kyep**
Simply deletes growth serum from the game.

Why? Because look at this mess:

🆑 Kyep
rscdel: Growth Serum removed from the game.
/🆑

### CODE REVIEW

**Comment by Anasari** [comment]
This PR have my full support because: 
1. Byond cannot scale-up sprite properly and as a result, scaling up a sprite is aesthetically unpleasing. This **cannot** be mitigated anyway from what meager knowledge I have on the matter
2. Scaling up sprite breaks the one-tile one sprite "rule" and lead to the sprite visually clipping into other tiles
3. Growth serum, unlike dwarfism is visually obnoxious and obstruct items
4. Growth serum do not have any mechanisms connected to it, unlike dwarfism and therefore is purely visual
5. Growth serum's effect of gigantism is just too silly for a MRP environment.

**Comment by Draconic-saint** [comment]
I prefer fethas's max cap over a plain removal, but i don't know how to refer to other pr's on mobile

**Comment by Kyep** [comment]
Simply put a thumbs up on whichever PR you prefer.

**Comment by jughu** [comment]
Honestly peoplr say its cosmetic but some people click your sprite to target you i once was a rev at maxgrowth cmo tried to tase me he clicked on my head and it still shot at the tile instead of me its basically miss target and your fucked serum

**Comment by Anasari** [comment]
What jughu said seems to be another reason to be in favour of removal instead of cap , in my opinion

**Comment by Fox-McCloud** [comment]
This is very reactionary---yes, it's annoying and problematic and needs to be addressed, but out and out removing it, when we can adjust how strong it is, seems rather non-sensical.

**Comment by jughu** [comment]
Just only make it 1.2 growth if it was cosmetic and not like 2.5

**Comment by uraniummeltdown** [comment]
# IMPROVE DON'T REMOVE

**Comment by Kyep** [comment]
In my past experience working with giant sprites, I found that they never really look good, and frequently have clicking/hitbox issues. Normally I wouldn't advocate for straight-up-removing a feature, but I do feel that this feature adds nothing to the game, looks terrible, and has mechanical problems. Plus, the majority of people seem to want it removed outright - not just a lower cap on the max growth size. Hence, a PR to remove it.

**Comment by Spacemanspark** [comment]
I'd rather just remove it.

**Comment by jughu** [comment]
Make it admin only for special "events"

**Comment by Spacemanspark** [comment]
I'd be fine with that.

**Comment by Birdtalon** [comment]
I'd be more in favour of a hard cap on the size you can reach rather than just removing content outright. See #6435

**Comment by Crazylemon64** [comment]
#6435

**Comment by Crazylemon64** [comment]
@kyep

**Comment by Anasari** [comment]
While the growth serum use has reduced a lot since its nerf, I still feel a need to remove it, as when I see it used, it is still extremely silly and immersion breaking to me, and still look out of place.

I'll use the same reasons I used to support this PR initially.

**Comment by tristan1333** [comment]
hehe I feede tee amanita to random in bar hehe make them big hehe very good hehe remove this please hehe

**Comment by SamHPurp** [comment]
I still support removing this via this PR.

Thank you @Crazylemon64 for re-opening this.

**Comment by Crazylemon64** [comment]
2017-02-28-132806_403x284_scrot

**Comment by Aurorablade** [comment]
still no need to remove it.

**Comment by Fox-McCloud** [comment]
If people can be smaller/dwarf sized with genetics, I see now reason why they shouldn't be allowed to be bigger/giant.

**Comment by Necaladun** [comment]
I haven't actually seen this since the 1.5x nerf, so I can't really say how silly it looks.

It also thus hasn't been used much, so I'm against removing it.

**Comment by Krausus** [comment]
> If people can be smaller/dwarf sized with genetics, I see now reason why they shouldn't be allowed to be bigger/giant.

I do: smaller people don't have their sprite spilling over onto adjacent turfs that they aren't actually on.

> I haven't actually seen this since the 1.5x nerf, so I can't really say how silly it looks.

**Comment by uraniummeltdown** [comment]
It's not made that wildly anymore, if we can have superfart we can have this, ss13 is a silly game at heart

**Comment by tigercat2000** [comment]
@uraniummeltdown I am going to keep refering to us as Vorestation 2.0 until this is removed.

**Comment by Kyep** [comment]
I still think that the oversized HoP in the above graphic looks terrible, and that growth serum overall is low-rp. Having the dwarf gene is not the same. People with dwarfism look OK. This looks terrible.

That said, I've barely seen this used since the previous nerf to it. I'd probably be more strongly in favor of removing it outright if people started using it again, though.

**Comment by tigercat2000** [comment]
@Kyep i support this but please for the love of god please don't argue for this on the basis of "its low-rp"- a goddamn game mechanic that makes you 1.25x larger isn't "low rp", like i said, literally vorestation uses it, and they're an hrp server
argue for it because it looks like shit and gives us a reputation closer to that of an erp server

**Comment by Aurorablade** [comment]
i don't associate this with ERP. do i don't get that. I still sya nerf it down one more tick

**Comment by Draconic-saint** [comment]
Thing is, human sprites have little top margin for growth inside their own 32x32 tile. Event the smallest stretching will probably have the sprite exceed the 1 tile... i wish this could fixed, but I don't know if it can

**Comment by Rumiluntti** [comment]
There is no way this can be made to look good.
This does not add anything worthwhile.
Nothing to improve, just a shitty gimmick.

Fox's argument is irrelevant at best. How does a dorf gene in genetics have anything to do with a sprite obliterating botany meme? Why the fuck should there be a giant serum that does exactly jack shit outside of eyesore just because there is a dorf gene that has mechanical value? What?

Just let it die, the entire thing seems to be in agony already. Look at that mess.

**Comment by Fox-McCloud** [comment]
The dwarf gene makes you smaller, visually, the growth serum gene makes you large, visually----both contort the sprite and makes it less quality than it originally was, and both have a mechanic function; one makes you easier to click, one makes you harder to click; they're both the same, *thematically*.

**Comment by Rumiluntti** [comment]
You can run on tables without climbing them with the dorf gene, growth serum doesn't offer anything but eyesore.

**Comment by uraniummeltdown** [comment]
Could tone it down to 1.3x maximum and make the intervals smaller

**Comment by Fox-McCloud** [comment]
The consensus on this was "improve, don't remove".

As in, if this is currently still a problem, then it should reduce the ratio more and not remove the chem, outright.

Closing.

**Comment by Krausus** [comment]
> The consensus on this was "improve, don't remove".

Uhh, what consensus are you talking about, exactly? This PR always had more support than the "improve, don't remove" alternative that got pushed through regardless. And, guess what? Turns out, you can't "improve" your way out of the limitations imposed by the size of our player sprites and the way BYOND scaling works; no matter how small you make the scaling, it *will* spill onto adjacent turfs, and it *will* look like a blurry mess.

**Comment by Fox-McCloud** [comment]
The Maintainers+Headmins

**Comment by Krausus** [comment]
Ah, see, just using "the consensus" by itself makes it sound like the community came to an agreement, when it's more like, "the consensus among the only people whose opinions matter when we decide they do". That's too wordy, of course, but "we have decided..." would do the trick.

**Comment by SamHPurp** [comment]
Improve a feature that adds nothing but memey shit, don't remove it.

Zzzzz, really?

**Comment by Spacemanspark** [comment]
memes
